docs/core/getting_started/more-resources.md

```diff
@@ -0,0 +1,41 @@
+
+# More Resources
```

Comment:
it seems like somewhere in this file it would make sense to point them to the Orchestration docs